php/ini.jinja

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 # vim: ft=jinja
-{# php.ini management macro. #}
-{% from "php/macro.jinja" import sls_block, serialize %}
+{#- php.ini management macro. #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/macro.jinja" import sls_block, serialize %}
+{%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
 
-{% macro php_ini(filename, opts={}, settings={}) %}
+{% macro php_ini(filename, tofs_lookup, opts={}, settings={}) %}
   file.managed:
     {{ sls_block(opts) }}
     - name: {{ filename }}
-    - source: salt://php/files/php.ini
+    - source: {{ files_switch( ['php.ini'], tofs_lookup ) }}
```

Comment:
```suggestion
    - source: {{ files_switch(['php.ini'],
                              tofs_lookup,
              ) }}
```

* Maintain layout consistency when using the `files_switch` macro.

Reply:
Done